hikari/users.py

```diff
@@ -511,6 +511,11 @@ def default_avatar_url(self) -> files.URL:
             discriminator=int(self.discriminator) % 5,
             file_format="png",
         )
+    
+    @property
+    def display_avatar_url(self) -> files.URL:
+        """Display avatar URL for this user."""
+        return self.avatar_url or self.display_avatar_url
```

Comment:
```suggestion
        return self.avatar_url or self.default_avatar_url
```

Is probably what you actually wanted here